src/exam/Sequence.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { withExamStore } from '../hocs';
+import Exam from './Exam';
+import { getExamAttemptsData, getProctoringSettings } from '../data';
+
+/**
+ * SequenceExamWrapper is the component responsible for handling special exams.
+ * It takes control over rendering exam instructions unless exam is started only if
+ * current sequence item is timed exam. Otherwise, renders any children elements passed.
+ * @param children - Current course sequence item content (e.g. unit, navigation buttons etc.)
+ * @returns {JSX.Element}
+ * @example
+ * <SequenceExamWrapper sequence={sequence} courseId={courseId}>
+ *   {sequenceContent}
+ * </SequenceExamWrapper>
+ */
+const SequenceExamWrapper = ({ children, ...props }) => {
```

Comment:
after refactoring, "SequenceExamWrapper" from "src/exc/Sequence.jsx" is not used. moved to "src/index.jsx"